lib/hammer_cli_katello/cv_import_export_helper.rb

```diff
@@ -22,13 +27,34 @@ def find_local_component_id(component_from_export)
       found_composite_version.first['id']
     end
 
+    # rubocop:disable Metrics/CyclomaticComplexity
+    def check_fs_perms
+      directory = options[:dirname]
+
+      begin
+        while  directory != "/"
+          dirmode = File.stat(directory).mode
+          has_apache_owner = File.stat(directory).uid == APACHE_UID
+          has_apache_group = File.stat(directory).gid == APACHE_GID
+          # rubocop:disable LineLength
+          raise _("Apache does not not have permissions to access #{directory}") unless (dirmode & OTHERS_EXECUTE).nonzero? || (dirmode & GROUP_EXECUTE && has_apache_group).nonzero? || (dirmode & USER_EXECUTE && has_apache_owner).nonzero?
```

Comment:
what is the max line length? we can split this up using `\` probably

Reply:
Updated.